readme.md

```diff
@@ -68,3 +68,41 @@ However, there are some implementation details to be aware of:
   `u32` is enough for all practical uses.
 * `str` is encoded as `(u64, &[u8])`, where the `u64` is the number of
   bytes contained in the encoded string.
+
+## Specification
+
+Bincode's format will eventually be codified into a specification, along with
+its configuration options and default configuration. In the meantime, here are
+some frequently asked questions regarding use of the crate:
+
+### Is Bincode suitable for storage?
+
+The encoding format is stable across minor revisions, provided the same
+configuration is used. This should ensure that later versions can still read
+data produced by a previous versions of the library if no major version change
+has occured. Beware that in the Rust ecosystem, for a `0.x` version, any change
+in `x` is considered a **major** version change.
```

Comment:
This last sentence is probably unnecessary since bincode is post 1.0

Reply:
Fixed, only took three months :)